docs/src/components/header.js

```diff
@@ -8,9 +8,7 @@ import CloseIcon from './icons/close-icon'
 import coa from '../images/coa_white.svg'
 import { Wrapper, PanelProvider } from './theme'
 import { External } from './link'
-
-const notifyBaseUrl =
-  process.env.GATSBY_NOTIFY_BASE_URL || 'https://notify.gov.au'
+import { NOTIFY_BASE_URL } from './core/env-vars'
```

Comment:
Just `notifyBaseURL`—don't need caps because it's just a variable.

Reply:
It's being used as a constant though. I've followed this convention for constants, not sure if the team wants to adhere to it: https://www.w3schools.com/js/js_conventions.asp

Reply:
Nope don't care about that and personally think it's often bad guidance. `notifyBaseURL` comes from an env var at the moment but it could come from runtime config. At the end of the day it's just a variable. Also by having only one use of it in url.js, it is easy to change anyway so no big deal I guess. Prefer `notifyBaseURL`. Even things like `MAX_SIZE`—just do `const maxSize = 10` and you're set.